### PR TITLE
Set iOS deployment target to iOS 13

### LINKED ISSUE #3369: Drop support for iOS 12

### PR DESCRIPTION
**Pull request by kylehickinson**
## Summary of Changes

This pull request fixes #3369

Test plan:
Make a basic sanity check the app works in general, can be done during manual verifications

Verify that NFC yubikey registration works, basic test only,
go to https://demo.yubico.com/webauthn-technical/registration and try to registar a NFC key

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: Client/Frontend/Browser/ContextMenuHelper.swift [hunk 1]
```diff
@@ -1,163 +0,0 @@
...
-    }
-}
-
-extension ContextMenuHelper: TabContentScript {
-    class func name() -> String {
-        return "ContextMenuHelper"
-    }
-
-    func scriptMessageHandlerName() -> String? {
-        return "contextMenuMessageHandler"
```
Comment (iccub) [inline Client/Frontend/Browser/ContextMenuHelper.swift:119]: Can we remove associated js file too? I believe it is `ContextMenu.js`

Comment (kylehickinson) [reply Client/Frontend/Browser/ContextMenuHelper.swift:119]: Yeah we can I think, let me do that

**Review by iccub — APPROVED** [review]
Developer morale to the moon 🚀 next stop at dropping 13.2 :D

**Review by soner-yuksel — APPROVED** [review]
Magnificent 🥇

**Pull request merged**